src/argilla_server/search_engine/commons.py

```diff
@@ -237,6 +237,19 @@ def _unify_sort_by_with_order(sort_by: List[SortBy], order: List[Order]) -> List
     return new_order
 
 
+def is_response_status_scope(scope: FilterScope) -> bool:
+    if not isinstance(scope, ResponseFilterScope):
+        return False
+
+    if not scope.property == "status":
+        return False
+
+    if scope.question is not None:
+        return False
+
+    return True
```

Comment:
Maybe working in positive logic and only using boolean operators is easier to understand:

```suggestion
def is_response_status_scope(scope: FilterScope) -> bool:
    return (
        isinstance(scope, ResponseFilterScope) and
        scope.property == "status" and 
        scope.question is None
    )
```